app/assets/javascripts/AngularConfig.js.coffee

```diff
@@ -26,7 +27,8 @@ angular.module('dahlia.controllers',[])
       views:
         'nav@':
           templateUrl: 'shared/templates/nav.html'
-          controller: 'NavController'
+        'nav-mobile@':
```

Comment:
Good question... I was just following the naming convention that @kaptron established.